internal/cli/up.go

```diff
@@ -30,14 +29,15 @@ func (c *upCmd) run(args []string) error {
 	}
 	defer proc.Kill()
 
-	dCli, err := tiltd_client.NewDaemonClient()
-	if err != nil {
-		return err
-	}
-	err = dCli.CreateService(ctx, "blahblahblah")
-	if err != nil {
-		return err
-	}
+	//dCli, err := tiltd_client.NewDaemonClient()
```

Comment:
👀

Reply:
...i know this is awkward but tbh i'm gonna leave it cuz it makes the code easier to test

Reply:
You could make a new `cmd/` binary to test and not commit if that's what you're looking for.